src/main/java/it/pagopa/ecommerce/commons/domain/TransactionActivated.java

```diff
@@ -56,7 +56,6 @@ public TransactionActivated(
                         paymentNotices,
                         email,
                         creationDate,
-                        status,
```

Comment:
Can we remove`TransactionStatusDto status` constructor parameter as it is no more used?

Reply:
Yep, missed! ty 🎉🙂